test/extensions/filters/network/common/fuzz/uber_writefilter.cc

```diff
@@ -0,0 +1,121 @@
+#include "test/extensions/filters/network/common/fuzz/uber_writefilter.h"
+
+#include "common/config/utility.h"
+#include "common/config/version_converter.h"
+
+using testing::_;
+using testing::Return;
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+void UberWriteFilterFuzzer::reset() {
+  // Reset some changes made by current filter on some mock objects.
```

Comment:
This comment is vague and talks about your implementation.  Instead, talk about the goal:  Reset the state of dependancies so that a new fuzz input starts in a clean state.

Reply:
> This comment is vague and talks about your implementation. Instead, talk about the goal: Reset the state of dependancies so that a new fuzz input starts in a clean state.

Fixed, thanks!